### ISSUE TITLE
[L4D1] Force servers to update

### ISSUE THREAD

**Issue by AshThe9thSurvivor** [ValveSoftware/Source-1-Games#3736]
In L4D1 the players are divided into different versions, and the version with more players is 1.0.3.1, this happens because the administrators of the community servers do not update, so I ask that they force them to update, as happens in TF2 , when an update comes out, a notification pops up that it is necessary to update so that players can connect, this should be done for L4D1.

I know you don't care, and what's more, you want to kill the game, but at least do that for our community.

**Comment by Diggingwolf2605** [comment]
They would have to make -autoupdate work with Windows based server's since the command is for Linux only right now. The update would have to be applied by the server owner manually if it is running on Windows. I've never ran into any server's that has an out of date build. Is there a specific server that would be the issue?

**Comment by AshThe9thSurvivor** [comment]
> They would have to make -autoupdate work with Windows based server's since the command is for Linux only right now. The update would have to be applied by the server owner manually if it is running on Windows. I've never ran into any server's that has an out of date build. Is there a specific server that would be the issue?

I'm surprised what you're asking me, I don't know if they know it, but most of the L4D1 community servers are in version 1.0.3.1, and currently the game is in version 1.0.4.0, for that same reason, all the players are changing their version to 1.0.3.1; For this reason, I request that you force everyone to update, that always when you get a new update, do not allow them to use your custom server if they are not in the latest version.

**Comment by Diggingwolf2605** [comment]
I don't think it would be necessary to force all clients to be on the latest build but that's just me. The game has other bugs/exploits which have been asked to be fixed for several years. I don't know why they didn't fix any of them with the latest 1.0.4.0 build. Hopefully they will address the issues with a Vulkan update and add better controller support for the game.

**Comment by AshThe9thSurvivor** [comment]
> I don't think it would be necessary to force all clients to be on the latest build but that's just me. The game has other bugs/exploits which have been asked to be fixed for several years. I don't know why they didn't fix any of them with the latest 1.0.4.0 build. Hopefully they will address the issues with a Vulkan update and add better controller support for the game.

Of course it is necessary, it is true that the game has errors, and big ones, but having the players separated into versions, and worst of all, that most of them are in those past versions is something very serious, to see if that would seem normal in CSGO for example, I'm sure they would go crazy if that happened, they just don't do it because it's L4D1, a game that they really want to see with population 0.
